CHANGELOG.md

```diff
@@ -42,6 +42,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Set `auto_mode: True` for the `ElevenLabsTTSService`. This change
+  significantly reduces the latency by disabling the chunk schedule and all
+  buffers.
```

Comment:
i agree to make this true by default but lets add it in input params to make it configurable if someone wants to exclude it

Reply:
Good point! I'll make that change in this PR.